docs/ldml/tr35-dates.md

```diff
@@ -1877,15 +1879,19 @@ Some of the examples are drawn from real data, while others are for illustration
            * "Pacific Time (Canada)" // for the zone Vancouver in the locale en_MX.
            * "Mountain Time (Phoenix)"
            * "Pacific Time (Whitehorse)"
-5. For the generic location format:
-   1. From the TZDB get the country code for the zone, and determine whether there is only one timezone in the country. If there is only one timezone or if the zone id is in the `<primaryZones>` list, format the country name with the _regionFormat_, and return it.
+5. For the location formats (generic or specific):
+   1. Get the _regionFormat_ format according to type (generic, standard, or daylight).
+   2. From the TZDB get the country code for the zone, and determine whether there is only one timezone in the country. 
+      1. If there is only one timezone or if the zone id is in the `<primaryZones>` list, continue with short country name, if it exists, otherwise the country name.
+      2. Otherwise, continue with the localized name of the exemplar city for the zone.
+   3. Format the region format obtained in step 1 with the location obtained in step 2.
       * Examples:
-        * Europe/Rome → IT → "Italy Time" // for English
-        * Asia/Shanghai → CN → "China Time" // Asia/Shanghai is the _primaryZone_ for China
-        * Africa/Monrovia → LR → "Hora de Liberja"
-        * America/Havana → CU → "Hora de CU" // if CU is not localized
-   2. Otherwise format the exemplar city with the _regionFormat_, and return it.
-      1. America/Buenos_Aires → "Buenos Aires Time"
+        * America/Buenos_Aires → "Buenos Aires Time" // multiple zones in AR
+        * Asia/Shanghai → "China Time" // Asia/Shanghai is the _primaryZone_ for CN
+        * Asia/Shanghai → "China Summer Time" // daylight 
```

Comment:
This all looks good, with one caveat. In the examples, it looks like Asia/Shanghai can arbitrarily produce two different values. I think there is an implicit additional parameter, which we can add to make it clear what is happening. (Might also reformat as a table in that case, for clarity, and to have the comment in a separate column.)

Reply:
done